CMakeLists.txt

```diff
@@ -28,6 +28,7 @@ option(SERVICE_PAM "Pam service" ON)
 option(SERVICE_GREETD "Greet service" ON)
 option(SERVICE_UPOWER "UPower service" ON)
 option(SERVICE_NOTIFICATIONS "Notification server" ON)
+option(INSTALL_QML_LIB "Insatlling the QML lib" OFF)
```

Comment:
I would install this by default. The only reason it isn't currently is I didn't spend much time on it, not an objection to it being in the package. Also stick it around the top of the option list.